app/models/user.rb

```diff
@@ -7,5 +7,5 @@ class User < ApplicationRecord
     format: {with: VALID_EMAIL_REGEX},
     uniqueness: {case_sensitive: false}
   has_secure_password
-  validates :password, presence: true, length: {minimum: Settings.user.min_password_length}
+  validates :password, length: {minimum: Settings.user.min_password_length}
```

Comment:
Vì nếu để presence: true khi ko điền password thì sẽ hiện thông báo password cant be blank 2 lần. Trong has_secure_password đã validate trường hợp này rồi nên mình bỏ đi :D